sockeye/utils.py

```diff
@@ -214,8 +214,7 @@ def update(self, labels, preds):
             self.num_inst += n
 
 
-def smallest_k(matrix: np.ndarray, k: int,
-               only_first_row: bool = False) -> Tuple[Tuple[np.ndarray, np.ndarray], np.ndarray]:
+def smallest_k(matrix: np.ndarray, k: int) -> Tuple[Tuple[np.ndarray, np.ndarray], np.ndarray]:
```

Comment:
can you also adapt smallest_k_mx to make them  equivalent?

Reply:
done